lib/mutant/meta/example/verification.rb

```diff
@@ -0,0 +1,86 @@
+module Mutant
+  module Meta
+    class Example
+      # Example verification
+      class Verification
+        include Adamantium::Flat, Concord.new(:example, :mutations)
+
+        # Test if mutation was verified successfully
+        #
+        # @return [Boolean]
+        def success?
+          missing.empty? && unexpected.empty? && no_diffs.empty?
```

Comment:
I noticed you seem to have completely dropped the `unparser` method and `unparser.empty?`. Was that performing a check that is no longer necessary?